nextcord/interactions.py

```diff
@@ -1252,3 +1257,43 @@ class InteractionMessage(_InteractionMessageMixin, Message):
     """
 
     pass
+
+
+class MessageInteraction:
+    """Represents a message's interaction data, regardless of application.
+
+    A message's interaction data is a property of a message when the message
+    is a response to an interaction from any bot.
+    .. versionadded:: 2.0
+
+    Attributes
+    -----------
+    id: :class:`int`
+        The interaction's ID.
+    type: :class:`InteractionType`
+        The interaction type.
+    name: :class:`str`
+        The application ID that the interaction was for.
+    user: :class:`User`
```

Comment:
The Discord docs call this `member` and it is optional

https://discord.com/developers/docs/interactions/receiving-and-responding#message-interaction-object-message-interaction-structure

Reply:
I'm not sure how to combine the message's guild with the member from the interaction